core/domain/question_fetchers_test.py

```diff
@@ -114,16 +127,18 @@ def test_cannot_get_question_from_model_with_invalid_schema_version(self):
         all_question_models = question_models.QuestionModel.get_all()
         self.assertEqual(all_question_models.count(), 0)
 
-        question_id = question_services.get_new_question_id()
+        question_id = question_services.get_new_question_id() # type: ignore[no-untyped-call]
 
-        question_model = question_models.QuestionModel(
+        question_model: Optional[
```

Comment:
Why is this `Optional` ?

Reply:
question_model can be Optional so mypy throws this error when Optional is not used: 
![Screen Shot 2022-06-12 at 9 59 21 PM](https://user-images.githubusercontent.com/85202273/173266299-42dc061f-5b3d-4ea7-97f9-d3e7985d23b0.png)

Reply:
I think that error pops up because we are trying to change the type of `question_model` in line 153.
You can maybe rename the variable in line 153 because that `question_model` is `Optional` since it's fetched.

Reply:
Done